src/pages/runs.tsx

```diff
@@ -0,0 +1,25 @@
+import { formatDate } from "date-fns";
+import { toZonedTime } from "date-fns-tz";
+import { navigate, PageProps } from "gatsby";
+import React, { useEffect } from "react";
+import { Helmet } from "react-helmet";
+
+const RunsPage: React.FC<PageProps<null>> = () => {
+  const currentDate = formatDate(
+    toZonedTime(new Date(Date.now()), "UTC"),
+    "yyyy-MM-dd"
+  );
+
+  useEffect(() => {
```

Comment:
You needn't use `useEffect` here, not that it matters because it'll be changing later but yeah.

Check this out if you haven't, it's really cool, i just found it recently - https://react.dev/learn/you-might-not-need-an-effect

Reply:
fixed to using Redirect component instead